remap-gff3/bin/remap-gff3.py

```diff
@@ -78,7 +78,7 @@ def filter_not_exact_match(CrossMap_mapped_file, CrossMap_log_file, filtered_fil
                             if attribute_dict['ID'] in ID_set:
                                 logger.error('All features should have an unique ID. Please run the command with -tmp_ID.')
                                 sys.exit(0)
-                            else:
+                            elif not 'not exact match' in match_state:
```

Comment:
Although the syntax have the same effect of `elif not 'not exact match' in match_state`, it seems to me that using `if item not in itemlist` will be better. Similar syntax is explained in PEP8 https://www.python.org/dev/peps/pep-0008/.

Reply:
Why `if item not in itemlist` is better?

Reply:
The "better" means more readable here. See error code 713 in  pep8 python style guide checker.
http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes